e2etests/tests/receiving.go

```diff
@@ -292,6 +327,111 @@ var _ = ginkgo.Describe("Receiving routes", func() {
 						ValidateNodesDoNotHaveRoutes(pods, p.FRR, []string{"fc00:f853:ccd:e799::/64", "fc00:f853:ccd:e800::/64"}...)
 					}
 				},
+				splitCfg: splitByNeigh,
+			}),
+		)
+
+		ginkgo.DescribeTable("Multiple configs", func(p params) {
```

Comment:
can we add a test where we receive on one side and advertise on the other? THis is gonna be our primary use case together with metallb

Reply:
added (in a different file)